### PR TITLE
Fixes to docstring formatting

### PR DESCRIPTION
**Pull request by jkgoodrich**
These are just the easy fixes to docstrings reported by `pydocstyle`. The remaining problems are actual missing docstrings and will take more thought to add.

### CODE REVIEW

File: gnomad/assessment/sanity_checks.py [hunk 1]
```diff
@@ -110,8 +110,9 @@ def sample_sum_check(
     sort_order: List[str] = SORT_ORDER,
 ) -> None:
     """
-    Compute afresh the sum of annotations for a specified group of annotations, and compare to the annotated version;
-    display results from checking the sum of the specified annotations in the terminal.
+    Compute the sum of annotations for a specified group of annotations, and compare to the annotated version.
+
+    Results from from checking the sum of the specified annotations are printed in the terminal.
```
Comment (nawatts) [inline gnomad/assessment/sanity_checks.py:115]: "from from"

File: gnomad/sample_qc/pipeline.py [hunk 2]
```diff
@@ -27,8 +27,10 @@ def filter_rows_for_qc(
     snv_only: bool = True,
 ) -> hl.MatrixTable:
     """
-    Annotates rows with `sites_callrate`, `site_inbreeding_coeff` and `af`, then applies thresholds.
-    AF and callrate thresholds are taken from gnomAD QC; inbreeding coeff, MQ, FS and QD filters are taken from GATK best practices
+    Annotate rows with `sites_callrate`, `site_inbreeding_coeff` and `af`, then applies thresholds.
```
Comment (nawatts) [inline gnomad/sample_qc/pipeline.py:30]: For imperative mood throughout the description, maybe change "applies" to "apply".

File: gnomad/sample_qc/relatedness.py [hunk 3]
```diff
@@ -51,7 +51,8 @@ def get_duplicated_samples(
     rel_col: str = "relationship",
 ) -> List[Set[str]]:
     """
-    Given a pc_relate output Table, extract the list of duplicate samples. Returns a list of set of samples that are duplicates.
+    Extract the list of duplicate sample using a Table ouput from pc_relate.
```
Comment (nawatts) [inline gnomad/sample_qc/relatedness.py:54]: "sample" => "samples"

File: gnomad/sample_qc/relatedness.py [hunk 4]
```diff
@@ -285,8 +296,12 @@ def group_parent_child_pairs_by_fam(
         parent_child_pairs: Iterable[Tuple[str, str]]
     ) -> List[List[Tuple[str, str]]]:
         """
+        Group parent-child pairs into a list of families.
+
         Takes all parent-children pairs and groups them by family.
```
Comment (nawatts) [inline gnomad/sample_qc/relatedness.py:301]: Should this line be removed? It's nearly a duplicate of the first line.

File: gnomad/sample_qc/relatedness.py [hunk 5]
```diff
@@ -262,8 +268,13 @@ def infer_families(
     relationship_col: str = "relationship",
 ) -> hl.Pedigree:
     """
-    This function takes a hail Table with a row for each pair of individuals i,j in the data that are related (it's OK to have unrelated samples too).
-    The `relationship_col` should be a column specifying the relationship between each two samples as defined in this module's constants.
+    Generate a pedigree containing trios inferred from the `relationship_ht`.
+
+    This function takes a hail Table with a row for each pair of individuals i,j in the data that are
+    related (it's OK to have unrelated samples too).
```
Comment (nawatts) [inline gnomad/sample_qc/relatedness.py:274]: Maybe "each pair of related individuals i, j in the data" vs "each pair of individuals i,j in the data that are related"?

File: gnomad/sample_qc/relatedness.py [hunk 6]
```diff
@@ -391,7 +409,9 @@ def get_children(possible_parents: Tuple[str, str]) -> List[str]:
 
         def check_sibs(children: List[str]) -> bool:
             """
-            3. If there are multiple children for a given parent pair, all children should be siblings with each other
+            Confirm that all children a parent pair are siblings with each other.
```
Comment (nawatts) [inline gnomad/sample_qc/relatedness.py:412]: "all children of a parent pair"

File: gnomad/sample_qc/relatedness.py [hunk 7]
```diff
@@ -547,10 +569,12 @@ def create_fake_pedigree(
     real_pedigree: Optional[hl.Pedigree] = None,
 ) -> hl.Pedigree:
     """
-    Generates a pedigree made of trios created by sampling 3 random samples in the sample list.
-    If `real_pedigree` is given, then children the resulting fake trios will not include any trio with proband - parents that are in the real ones.
-    Each sample can be used only once as a proband in the resulting trios.
-    Sex of probands in fake trios is random.
+    Generate a pedigree made of trios created by sampling 3 random samples in the sample list.
+
+    - If `real_pedigree` is given, then children the resulting fake trios will not include any trio with proband - parents
```
Comment (nawatts) [inline gnomad/sample_qc/relatedness.py:574]: "children in the resulting"

File: gnomad/utils/gen_stats.py [hunk 8]
```diff
@@ -66,7 +68,9 @@ def add_stats(
         i: hl.expr.StructExpression, j: hl.expr.StructExpression
     ) -> hl.expr.StructExpression:
         """
-        This merges two stast counters together. It assumes that all stats counter fields are present in the struct.
+        Merge two stast counters together.
```
Comment (nawatts) [inline gnomad/utils/gen_stats.py:71]: Is this supposed to be "stats counters"?

File: gnomad/utils/liftover.py [hunk 9]
```diff
@@ -28,16 +28,15 @@ def get_liftover_genome(
     t: Union[hl.MatrixTable, hl.Table]
 ) -> Tuple[hl.genetics.ReferenceGenome, hl.genetics.ReferenceGenome]:
     """
-    Infers reference genome build of input data and assumes destination reference genome build. 
+    Infer reference genome build of input data and assumes destination reference genome build.
```
Comment (nawatts) [inline gnomad/utils/liftover.py:31]: "assumes" => "assume"

File: gnomad/utils/plotting.py [hunk 10]
```diff
@@ -344,7 +347,8 @@ def plot_hail_file_metadata(
     t_path: str,
 ) -> Optional[Union[Grid, Tabs, bokeh.plotting.Figure]]:
     """
-    Takes path to hail Table or MatrixTable (gs://bucket/path/hail.mt), outputs Grid or Tabs, respectively.
+    Take path to hail Table or MatrixTable (gs://bucket/path/hail.mt), outputs Grid or Tabs, respectively.
```
Comment (nawatts) [inline gnomad/utils/plotting.py:350]: "outputs" => "output"

File: gnomad/variant_qc/evaluation.py [hunk 11]
```diff
@@ -20,8 +20,8 @@ def compute_ranked_bin(
     n_bins: int = 100,
     desc: bool = True,
 ) -> hl.Table:
-    """
-    Returns a table with a bin for each row based on the ranking of `score_expr`.
+    r"""
```
Comment (nawatts) [inline gnomad/variant_qc/evaluation.py:23]: Is a raw string necessary here?

File: gnomad/variant_qc/pipeline.py [hunk 12]
```diff
@@ -38,8 +38,9 @@ def create_binned_ht(
     add_substrat: Optional[Dict[str, hl.expr.BooleanExpression]] = None,
 ) -> hl.Table:
     """
-    This is meant as a default wrapper for `compute_ranked_bin`. It annotates table with a bin, where variants are
-    binned based on score into `n_bins` equally-sized bins.
+    Annotate each row of `ht` with a bin based on the score annotation into `n_bins` equally-sized bins.
```
Comment (nawatts) [inline gnomad/variant_qc/pipeline.py:41]: I think this sentence is missing a verb. "based on binning" ?

**Review by nawatts — APPROVED** [review]
Thanks for doing this!

A few minor comments on some docstrings.

File: gnomad/variant_qc/evaluation.py (hunk 11 above)
Comment (jkgoodrich) [reply gnomad/variant_qc/evaluation.py:23]: It complained about it because of the `\` needed below for formatting the table

**Pull request merged**